hr_expense_cancel/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2019 Tecnativa - Ernesto Tejeda
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Hr expense cancel",
+    "version": "11.0.1.0.0",
+    "author": "Tecnativa, "
+              "Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/hr",
+    "license": "AGPL-3",
+    "category": "Human Resources",
+    "depends": [
+        'hr_expense',
+        'account_cancel',
```

Comment:
Why depending on this one if you are not checking the flag in the journal?

Reply:
The objective of this module is to give the possibility of canceling the expense sheet and this is not possible if you can't cancel 'journal entries' and the account_cancel module must be installed to be able to cancel 'journal entries'. That is, if the account_cancel module is not installed then the expense sheets can not be canceled. So, I think that in order to use this module, account_cancel must also be installed.

Reply:
OK, technical dependencies are not put usually here, but let's don't extend more time in this for now.